examples/order_book/01_create_market.py

```
@@ -0,0 +1,127 @@
+        for market in markets:
+            if market.get('settle_time') == settle_timestamp:
+                query_id = market.get('id')
+                print(f"\n✓ Found our market!")
+                print(f"  Query ID: {query_id}")
+                print(f"  Hash: {market.get('hash', b'').hex() if isinstance(market.get('hash'), bytes) else market.get('hash', 'N/A')}")
+                print(f"  Settle Time: {market.get('settle_time')}")
+                print(f"  Settled: {market.get('settled')}")
+                break
+
+        if query_id:
+            print(f"\n✓ Market created successfully!")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)